### PR TITLE
Server clean up

### PR DESCRIPTION
**Pull request by jburel**
Fix javadoc warning in server component
the following warning (not under our control) should remain
```

/PATHTOREPO/repo/openmicroscopy/components/server/target/libs/subethasmtp.jar(org/subethamail/smtp/server/SMTPServer.class): warning: Cannot find annotation method 'value()' in type 'GuardedBy'
```
--no-rebase

### CODE REVIEW

**Comment by sbesson** [comment]
Fix for the remaining warning proposed in https://github.com/jburel/openmicroscopy/pull/9

File: components/server/test/ome/server/itests/PojosDaoTest.java [hunk 1]
```diff
@@ -28,7 +25,7 @@
 import org.testng.annotations.Test;
 
 /**
- * tests for an up-and-coming pojos data access TODO rename
+ * tests for an up-and-coming pojos data access
  * "PojosQuerySourceTest"
```
Comment (mtbc) [inline components/server/test/ome/server/itests/PojosDaoTest.java]: is this line part of the removed TODO?

File: components/server/src/ome/services/eventlogs/EventBacklog.java [hunk 2]
```diff
@@ -139,7 +137,8 @@ public synchronized EventLog remove() {
     /**
      * Flips the {@link EventBacklog} to the "adding" state if it is empty,
      * otherwise to the "removing" state. This is necessary since the indexing
-     * happens only at flush time in the {@link FullTextIndexer}. This
+     * happens only at flush time in the
+     * {@link ome.services.fulltext.FullTextIndexer}.This
```
Comment (mtbc) [inline components/server/src/ome/services/eventlogs/EventBacklog.java]: space after period

File: components/server/src/ome/services/fulltext/BridgeHelper.java [hunk 3]
```diff
@@ -51,16 +44,15 @@
...
-     * 
-     * @DEV.TODO add to constants
      */
+    //TODO add to constants
     public final static String COMBINED = "combined_fields";
 
     /**
      * Simpler wrapper to handle superclass proxy objects (e.g. Annotation)
-     * which do * not behave properly with instanceof checks.
+     * which do * not behave properly with instance of checks.
```
Comment (mtbc) [inline components/server/src/ome/services/fulltext/BridgeHelper.java]: "instanceof" is correct

File: components/server/src/ome/services/graphs/GraphOpts.java [hunk 4]
```diff
@@ -96,7 +97,7 @@
      *
      * @param op Current {@link Op} to add to the stack
      * @param modified Whether or not the value was changed by the user
-     * @param details Active user login
+     * @param ec The event context.Active user login
```
Comment (mtbc) [inline components/server/src/ome/services/graphs/GraphOpts.java]: did you mean to keep the "Active user login"? If so, maybe adjust spacing/punctuation

File: components/server/src/ome/services/fulltext/BridgeHelper.java (hunk 3 above)
Comment (jburel) [inline components/server/src/ome/services/fulltext/BridgeHelper.java]: this was fixed by another commit

Comment (mtbc) [inline components/server/src/ome/services/fulltext/BridgeHelper.java]: ah, a different PR?

Comment (jburel) [inline components/server/src/ome/services/fulltext/BridgeHelper.java]: my bad, was looking at the wrong file

**Comment by jburel** [comment]
@mtbc: done

**Comment by mtbc** [comment]
Fixes look good. :+1:

**Comment by jburel** [comment]
Re-listed for tomorrow but probably not required.

**Comment by sbesson** [comment]
Merging. Thanks.

**Pull request merged**